Change UInt64-Base10 (#99)
ripple/opensource.ripple.com: docs/xls-33d-multi-purpose-tokens/reference/mptokenissuance.md

| `Flags`             | number    | UInt32        | See [MPTokenIssuance Flags](#mptokenissuance-flags) |
| `Issuer`           | string    | AccountID     | The address of the account that controls both the issuance amounts and characteristics of a particular fungible token. |
| `AssetScale` | number    | UInt8       | An asset scale is the difference, in orders of magnitude, between a standard unit and a corresponding fractional unit. More formally, the asset scale is a non-negative integer (0, 1, 2, …) such that one standard unit equals 10^(-scale) of a corresponding fractional unit. If the fractional unit equals the standard unit, then the asset scale is 0. |

> | `MaximumAmount`         | string    | Base10        | This value is an unsigned number that specifies the maximum number of MPTs that can be distributed to non-issuing accounts (i.e., minted). For issuances that do not have a maximum limit, this value should be set to 18,446,744,073,709,551,615. |

Reviewer: ```suggestion
| `MaximumAmount`         | string    | UInt64        | This value is an unsigned number that specifies the maximum number of MPTs that can be distributed to non-issuing accounts (i.e., minted). For issuances that do not have a maximum limit, this value should be set to 9,223,372,036,854,775,807. |
```

should also say that its base 10 in the description

> | `OutstandingAmount` | string    | Base10        | Specifies the sum of all token amounts that have been minted to all token holders. This value can be stored on ledger as a default type so that when its value is 0 it takes up less space on ledger. This value is increased whenever an issuer pays MPTs to a non-issuer account, and decreased whenever a non-issuer pays MPTs into the issuing account. |

Reviewer: ```suggestion
| `OutstandingAmount` | string    | Uint64        | Specifies the sum of all token amounts that have been minted to all token holders. This value can be stored on ledger as a default type so that when its value is 0 it takes up less space on ledger. This value is increased whenever an issuer pays MPTs to a non-issuer account, and decreased whenever a non-issuer pays MPTs into the issuing account. |
```

should also say that its base 10 in the description

> | `LockedAmount`      | string    | Base10        | A positive amount of tokens that are currently held in a token holder's account, but are unavailable to be used by the token holder. Locked tokens might, for example, represent value currently being held in escrow, or value that is otherwise inaccessible to the token holder. This value is stored as a default value such that its initial value is 0, in order to save space on the ledger for an empty MPT holding. |

Reviewer: ```suggestion
| `LockedAmount`      | string    | UInt64        | A positive amount of tokens that are currently held in a token holder's account, but are unavailable to be used by the token holder. Locked tokens might, for example, represent value currently being held in escrow, or value that is otherwise inaccessible to the token holder. This value is stored as a default value such that its initial value is 0, in order to save space on the ledger for an empty MPT holding. |
```

should also say that its base 10 in the description

> | `LockedAmount`      | string    | UInt64        | A positive amount of tokens that are currently held in a token holder's account, but are unavailable to be used by the token holder. Locked tokens might, for example, represent value currently being held in escrow, or value that is otherwise inaccessible to the token holder. This value is stored as a default base 10 value such that its initial value is 0, in order to save space on the ledger for an empty MPT holding. |

Reviewer: `LockedAmount` has been removed everywhere. Haven't updated the spec yet, but might be convenient to remove this field in the PR while we are at it.
